modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -277,3 +277,28 @@ def __dump_netfilter_settings(node):
         Reads current netfilter settings on the node for debugging purposes.
         """
         return str(node.account.ssh_client.exec_command("sudo iptables -L -n")[1].read(), sys.getdefaultencoding())
+
+    def restart(self, timeout_sec=180):
+        """
+        Restart ignite cluster without cleaning.
+        """
+        self.stop()
+
+        for node in self.nodes:
+            super().start_node(node)
```

Comment:
why super()?

Reply:
so as not to call the IgniteAwareService.init_persistent(node)